### ISSUE TITLE
[REQUEST] Status Page Incident Discord Notifications

### ISSUE THREAD

**Issue by OnyxJeff** [louislam/uptime-kuma#1102]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

New Notification, UI Feature

### 🔖 Feature description

Is there anyway to add notifications to incidents on the status page?

### ✔️ Solution

In my use-case, I have individual monitors posting notifications to "internal" and "external" channels on Discord. It would be nice to also post the Incidents I set on the status page automatically as well on Discord.

**Comment by rasos** [comment]
I think that should be a generic option for any notification gateway (not just for Discord - maybe rephrase issue title).

Yes please offer  a checkbox at each notification: 

[ ] also send manually entered incident messages

**Comment by CommanderStorm** [comment]
The main discussion about public status page subscriptions happened here https://github.com/louislam/uptime-kuma/issues/916

**Comment by CommanderStorm** [comment]
@rasos 
Filtering notifications is tracked here https://github.com/louislam/uptime-kuma/issues/508
